packages/build/package.json

```diff
@@ -130,27 +130,22 @@
     "@opentelemetry/api": "~1.8.0",
     "@opentelemetry/sdk-trace-base": "~1.24.0",
     "@types/node": "^14.18.53",
-    "@vitest/coverage-v8": "^0.33.0",
     "atob": "^2.1.2",
     "ava": "^4.0.0",
     "c8": "^7.12.0",
     "copyfiles": "^2.4.1",
     "cpy": "^9.0.0",
-    "fast-safe-stringify": "^2.0.7",
-    "get-bin-path": "^11.0.0",
     "get-node": "^12.0.0",
     "get-port": "^6.0.0",
-    "get-stream": "^6.0.0",
     "has-ansi": "^5.0.0",
     "moize": "^6.0.0",
     "npm-run-all2": "^5.0.0",
-    "path-key": "^4.0.0",
     "process-exists": "^5.0.0",
     "sinon": "^13.0.0",
     "tmp-promise": "^3.0.2",
     "tsd": "^0.31.0",
     "vitest": "^0.34.0",
-    "yarn": "^1.22.4"
+    "yarn": "^1.22.22"
```

Comment:
😓 This is used in a fixture used by a couple tests. I didn't see an easy way to improve it so I kept it.